### DISCUSSION TITLE
/dye rainbow for leather horse armor

### DISCUSSION THREAD

**Discussion by Sikatsu** [Zrips/CMI#2777] [Feature Requests]
It would be nice if /dye rainbow could be applied to leather horse armor.

**Comment by itsMarkiS** [comment]
oh yeah!! we need it

**Comment by CloudeLecaw** [comment]
does a leather horae armor exist ? i think not..
cmi cant create new items that doesnt exist...and if they do thwy must be dyeable too in vanilla like the leather armor

**Comment by Sikatsu** [comment]
It does exist

**Reply by CloudeLecaw** [reply]
> It does exist

but NOT dyeable in vanilla
